### PR TITLE
[frontend] fix filter display in retention policies

### PR DESCRIPTION
**Pull request by jpkha**
### Related issues

* https://github.com/OpenCTI-Platform/opencti/issues/5153

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [x] I wrote test cases for the relevant uses case
- [x] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: opencti-platform/opencti-front/src/components/filters/ReadonlyFiltersInLine.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    filterIconButtonContentQuery,
+    {
+      filters: removeIdFromFilterObject(
+        filters,
+      ) as unknown as GqlFilterGroup,
+    },
+  );
+  return (
+    <>
+      <>
```
Comment (labo-flg) [inline opencti-platform/opencti-front/src/components/filters/ReadonlyFiltersInLine.tsx]: double fragment ?

**Comment by Archidoit** [comment]
bad display of the 'internal mode' in stream, trigger, taxii, etc: 
image

**Comment by Archidoit** [comment]
We don't see anything in the Retention Policies list: 
image

**Comment by Archidoit** [comment]
Go in Custom Dashboard.
Create a widget of type Knowledge.
Add a filter for the 'source' (filter in orange).
An error occurs.

**Comment by Archidoit** [comment]
bad display of the trigger of a user (in Settings>Security>select a user>look at his triggers) : 
image

image

**Comment by Archidoit** [comment]
in triggers list, bad display if there are api filters and classic filters 
image

**Comment by Archidoit** [comment]
color of the global mode is always grey in widget (rather display it in orange/green for target and source filters ?) 
image

**Comment by Archidoit** [comment]
Dans les display de ligne de trigger/etc, quand je passe la souris sur un opérateur, ma souris se met en mode 'clickable' alors qu'on ne peut pas switcher d'opérateur
image

**Review by Archidoit — APPROVED** [review]

**Pull request merged**